sentry_sdk/tracing.py

```diff
@@ -213,6 +217,15 @@ def start_child(self, **kwargs):
         trace id, sampling decision, transaction pointer, and span recorder are
         inherited from the current span/transaction.
         """
+        hub = self.hub or sentry_sdk.Hub.current
+        client = hub.client
+
+        instrumenter = kwargs.get("instrumenter", INSTRUMENTER.SENTRY)
+        configuration_instrumenter = client and client.options["instrumenter"]
+
+        if instrumenter != configuration_instrumenter:
```

Comment:
why is this check duplicated here? imo doing it in hub.py is sufficient?

Reply:
As discussed I will leave this check here, to disable all manual instrumentation when the otel instrumenter is selected.